dataplanes/vpp/yaml/vpp-daemonset.yaml

```diff
@@ -28,7 +28,7 @@ spec:
           securityContext:
             privileged: true
           image: networkservicemesh/vpp:latest
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
If you rebase on master, I believe you will find this changes has already been made :) And some additional changes needed for the bind mounts to work correctly.